index.html

```diff
@@ -0,0 +1,125 @@
+            </div>
+            <div class="venue">
+                <h2 class="time">2023.5.12(FRI) ~ 13(SAT)</h2>
+                <p class="place">@Seattle, Washington</p>
+            </div>
+        </section>
+    </section>
+        <section class="programs">
+            <div class="heading2">
+               <h2 class="heading3">Main Program</h2>
+               <hr class="line">
+            </div>
+            <div class="program">
+              <div class="program-l">
+                <img class="proimg" src="./images/speech.svg">
+                <h3 class="proheading">Keynote speeches</h3>
+                <p class="propara">Keynote speeches from experts in the VR industry can provide attendees with valuable insights into the latest trends, innovations, and challenges in the field.</p>  
+              </div>
+              <div class="program-l">
+                <img class="proimg" src="./images/panel.svg">
+                <h3 class="proheading">Panel discussions</h3>
+                <p class="propara">Panel discussions featuring experts in various aspects of VR can provide attendees with a deeper understanding of specific topics and allow them to hear different perspectives and insights.</p>               
+              </div>
+              <div class="program-l">
+                <img class="proimg" src="./images/training.jpg">
+                <h3 class="proheading">Workshops</h3>
+                <p class="propara">Workshops and training sessions can provide attendees with hands-on experience with VR technology and allow them to develop practical skills and knowledge.</p>                
+              </div>
+              <div class="program-l">
+                <img class="proimg" src="./images/vr.png">
+                <h3 class="proheading">VR experiences</h3>
+                <p class="propara">VR experiences and demonstrations can give attendees the opportunity to try out the latest VR hardware and software and experience immersive and interactive virtual environments.</p>               
+              </div>
+              <div class="program-l">
+                <img class="proimg" src="./images/networking.svg">
+                <h3 class="proheading">Networking events</h3>
+                <p class="propara">Networking events such as receptions, meet-and-greets, and informal gatherings can provide attendees with the opportunity to connect with other professionals in the VR industry, share ideas, and build relationships.</p>
+              </div>              
+            </div>
+            <div class="button-cintainer">
+            <button class="button">Join VR Odyssey 2023</button>
+           </div>
+           <div class="button-cintainer1">
+            <a class="button1" href="#">SEE THE WHOLE PROGRAM</a>
+           </div>
+        </section>
+        <section class="sponsers">
+          <h2 class="sponhead">Partners</h2>
+          <hr class="line4">
```

Comment:
- [ ]  You did a great job 😃 here at adding the line below the heading like the Behance, but I see that the color of this hr is different from the one in the Behance, so please 😊 try to put the correct color be like the Behance
- [ ]  Also  I see that the width of this hr is greater than the one in the Behance, so please 😊 try to decrease the width a little to be like the Behance 💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/225888404-6f8622e1-a32e-45c1-a1ef-a9623a95ae13.png)   |   ![image](https://user-images.githubusercontent.com/83360636/225888373-158350a3-c530-4632-bca3-3bf6787e1cc4.png)  |